vendor/auth_strategies/ldap_strategy/README.md

```diff
@@ -0,0 +1,64 @@
+## LDAP strategy for authentication
+
+This strategy is used to authenticate users using a set of configurable LDAP providers.
+
+The LDAP providers can be configured in `config/config.yml`. There is an example in
+`config/config.yml.example`. For each site there is a `ldap_username` and `ldap_password`
+and a list of configurations including:
+* The `host`, `port` and `domain` of a LDAP service
+* `authentication_query`. This string be used to authenticate the user in the LDAP service.
+The data introduced as user identifier (usually an email) will be interpolated in the
+query replacing the `%{user_identifier}` part.
+* `password_field`: The attribute used to store the password in LDAP
+* `email_field` and `name_field`. The attributes of LDAP used to create a new user in case
+it does not exist in Gobierto.
+
+If a user enters authentication data the strategy goes through the services until a service
+can validate the user. Then the strategy gets the email and user name from the service and
+uses them to find or create the Gobierto user. This strategy mantains the default gobierto
+authentication strategy, with email and password, so if a user is not found in LDAP have
+access with a valid email and password combination. To prevent this the password mechanism
+can be disabled on the strategy configuration (see below)
+
+### Install
+
+To make available from site configuration admin part, add to `auth_modules` in
+`config/application.yml`:
+
+```ruby
+    -
+      name: ldap_strategy
+      description: LDAP strategy
+      session_form: LdapSessionForm
+      password_enabled: true
+      domains: ['site1.gobierto.test', 'site2.gobierto.test']
```

Comment:
Are these ldap domains? I'd change the example domain to `site1.ldap.test` to avoid confusion.